traffic_prophet/countmatch/tests/test_neighbour.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import sklearn.metrics as sklm
+import sklearn.neighbors as skln
+
+from .. import neighbour as nbr
+
+from ...data import SAMPLE_LONLAT
+from ... import cfg
+
+
+class TestNeighbourLocalEuclidean:
+    """Tests Euclidean file-based neighbour finder.  Also tests base class."""
+
+    def setup(self):
+        # There are only 10 items, so this is an all-to-all comparison.
```

Comment:
"this is an all-to-all comparison" -> "this orders all other points by distance."